docs/02-guides/public-configuration.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: Read and extend public configurations
+description:
+  The public configuration is a set of configuration options that are available
+  to any user of the application, they are accessible both from the client and
+  the server.
+sidebar_position: 5
+---
+
+# Public Configuration
```

Comment:
**suggestion:** Makes this header match the documents title above

Reply:
It takes up to much visual space and makes the documentation harder to read, the "title" is implied by the two main sections
- "Extending the public configuration"
- "Reading the custom public configuration"

Have a look at side-by-side comparison

![image](https://github.com/front-commerce/developers.front-commerce.com/assets/39598117/c0d07431-e565-4624-8317-e680a8510974)

Reply:
Fair enough !